### PR TITLE
Update CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by Mohiit70**
Update Contributor Covenant Code of Conduct

updates the Contributor Covenant Code of Conduct with improvements to the language, clarity and also adds emojis for enhanced readability and positive tone throughout the document.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,75 +1,48 @@
...
 
-* The use of sexualized language or imagery, and sexual attention or
-  advances of any kind
-* Trolling, insulting or derogatory comments, and personal or political attacks
-* Public or private harassment
-* Publishing others' private information, such as a physical or email
-  address, without their explicit permission
-* Other conduct which could reasonably be considered inappropriate in a
-  professional setting
+* The use of sexualized language or imagery, and sexual attention or advances of any kind. 🚫❌🙅‍♀️
```
Comment (ShwetKhatri2001) [inline CODE_OF_CONDUCT.md:21]: Remove the emojis added at the end of sentences. It's not looking professional here.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -1,75 +1,48 @@
...
+This Code of Conduct applies within all community spaces and also applies when an individual is officially representing the community in public spaces. Examples of representing our community include using an official e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event.
 
-## Enforcement
+## 🔒 Enforcement
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported to the community leaders responsible for enforcement at
-gmail.
-All complaints will be reviewed and investigated promptly and fairly.
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported to the community leaders responsible for enforcement at [insert email address here]. All complaints will be reviewed and investigated promptly and fairly.
```
Comment (ShwetKhatri2001) [inline CODE_OF_CONDUCT.md:39]: Change this line - [insert email address here]

**Review by ShwetKhatri2001 — CHANGES_REQUESTED** [review]
Change the mentioned things, please.

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (Mohiit70) [reply CODE_OF_CONDUCT.md:39]: ok

> Change this line - [insert email address here]

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (ShwetKhatri2001) [reply CODE_OF_CONDUCT.md]: Do the same for all sentences.

**Comment by ShwetKhatri2001** [comment]
It's good to go now 👍 Merging it.

**Comment by ShwetKhatri2001** [comment]
Congrats on your first PR getting merged on AR-Webstore @Mohiit70  🚀

**Pull request merged**